src/app/pages/details/desktop-view/instructor-resource-tab/instructor-resource-tab.js

```diff
@@ -9,11 +9,26 @@ import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
 import {useUserStatus, usePartnerFeatures} from '../../common/hooks';
 import './instructor-resource-tab.css';
 
-function FreeStuff({heading, blurb}) {
+function FreeStuff({freeStuffContent, userStatus}) {
+    const blurbLookupByInstructorStatus = {
+        undefined: freeStuffContent.content,
+        true: freeStuffContent.contentLoggedIn,
+        false: <div className="blurb-body">
+            <FontAwesomeIcon icon="exclamation-circle" />{' '}
+            Your account must be instructor verified...
+            <a href="https://openstax.secure.force.com/help">Contact support</a>
```

Comment:
should this point to a specific help page about getting instructor verified instead of just `/help` ?